source/documentation/support/support.md

```diff
@@ -0,0 +1,34 @@
+## Troubleshooting 
```

Comment:
If you changed the name of the file to "support.md", shouldn't the document title be "Support"?

Also, why is the first header a h2 instead of a h1? Is it because multiple files build up to a single html?

Reply:
Yes, you're right re. the first point. 

On the second point, yes, it is indeed.